main.py

```diff
@@ -49,6 +49,7 @@ def parse_args():
                           help='auto temperature scheduling for the simulated annealing')
     parser.add_argument('-ts', '--start_temp', type=float, help='starting temperature for the simulated annealing')
     parser.add_argument('-te', '--end_temp', type=float, help='ending temperature for the simulated annealing')
+    parser.add_argument('-am', '--auto_meth', type=str, default='none', help='method for auto-temperature scheduling')
```

Comment:
My next pull request will add constraints to the values of arguments. It's suppose its okay for now, but we should be letting the users know what methods are available.
```suggestion
    parser.add_argument('-am', '--auto_meth', type=str, default='none', choices=('none', 'frame', 'factor', 'const'), help='method for auto-temperature scheduling')
```